examples/user-notification.amp.html

```diff
@@ -229,6 +229,37 @@
      <button on="tap:amp-user-notification3.dismiss">I accept</button>
   </amp-user-notification>
 
+  <amp-user-notification
+          layout=nodisplay
+          id="amp-user-notification4"
+          always-show
+          data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
Hello  Team,
Please remove my input into "amphtml"
Thank you
Wojtek 

    On Tuesday, February 23, 2016 2:53 PM, Rudy Galfi <notifications@github.com> wrote:
 

 In examples/user-notification.amp.html:> @@ -229,6 +229,37 @@
>       <button on="tap:amp-user-notification3.dismiss">I accept</button>
>    </amp-user-notification>
>  
> +  <amp-user-notification
> +          layout=nodisplay
> +          id="amp-user-notification4"
> +          always-show
> +          data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
Can't we have always-show be mutually exclusive to the show-ifs as enforced by the validator?—
Reply to this email directly or view it on GitHub.